webpack/assets/javascripts/react_app/components/CPUCoresInput/CPUCoresInput.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { WarningTriangleIcon } from '@patternfly/react-icons';
+import PropTypes from 'prop-types';
+import { HelpBlock } from 'patternfly-react';
+import NumericInput from '../common/forms/NumericInput';
+import './cpuCoresInput.scss';
+
+const CPUCoresInput = ({ label, maxValue }) => {
+  const [value, setValue] = useState(1);
+  return (
+    <div className="form-group">
```

Comment:
You should be able to import React components from `patternfly-react` instead of using `div` for `form-group` and `col-md-10`:
- `FormGroup`
    - https://5c507d49471426000887a6a7--react-bootstrap.netlify.app/components/forms/
- `Col`
    - https://patternfly-react-pf3.surge.sh/?path=/story/patternfly-react-layouts-grid--grid
     - https://5c507d49471426000887a6a7--react-bootstrap.netlify.app/layout/grid/